components/MobileNav.tsx

```diff
@@ -107,3 +105,46 @@ const MobileNav = () => {
 }
 
 export default MobileNav
+
+interface RenderMobileNavLinkProps extends NavItem {
```

Comment:
Build error:

```
Type error: An interface can only extend an object type or intersection of object types with statically known members.
  107 | export default MobileNav
  108 |
> 109 | interface RenderMobileNavLinkProps extends NavItem {
      |                                            ^
  110 |   clickFunc: () => void
  111 |   navLink: NavItem
  112 | }
Error: Command "yarn run build" exited with 1
```

I think you can use a type alias instead.

Reply:
Resolved this by switching to a `type` and dropping `extends NavItem`